### ISSUE TITLE
Special Notice Emails for Pilgrims w/passports <6 months or national origin issues

### ISSUE THREAD

**Issue by chris206** [206Tours/DB#1620]
Hey Brad,

Can we generate 3 types of emails intermittently to clients as follows:

Type 1: Advises clients that we need their passport information
IF they have incomplete passport info listed (passport#, DOB, passport exp)

Type 2: Advise clients that we need their passport information
IF they have passports expiring <6 months after return date

Type 3: Advise of the need to contact the consulates of each nation to ensure that visa requirements are understood.
IF there is a Pilgrim with national origin other than the US, Canada, UK, or Australia

**Comment by bradatkins** [comment]
#2 is done; however, I haven't enabled it yet as I'm not sure what verbage you want to include.

Put your text in the following section and let me know. Then I will test it further and we can enable it afterwards

![image](https://cloud.githubusercontent.com/assets/8440255/21484956/3e91f6da-cb69-11e6-943d-be436977c38d.png)

**Comment by bradatkins** [comment]
#1 is done, same thing as above...

**Comment by bradatkins** [comment]
#3 is done too. FYI these messages are located here:

![image](https://cloud.githubusercontent.com/assets/8440255/21486514/e7bcf65a-cb83-11e6-9e1b-a552ae709db0.png)

**Comment by chris206** [comment]
i dont have access to this stuff. I took a look and really we just need the verbiage.

Once I have it I will send it to you and we can activate it

**Comment by chris206** [comment]
Can you confirm if these emails are tagged, activated and being sent?

If it has, can you provide an order number as an example for each message so that I can review it?

**Comment by bradatkins** [comment]
They are not being sent because I'm waiting for someone to put in the text for the emails:

![image](https://cloud.githubusercontent.com/assets/8440255/23474538/14d28516-fe82-11e6-9514-a1843e58822b.png)

**Comment by chris206** [comment]
ok, like a dope I just checked the first one and saw text entered. Can we activate this one if tagged?
I will ask Sandy about the other 2

**Comment by chris206** [comment]
For some reason, even though I am currently an "Admin" - I am unable to modify any info. I've copied and pasted the message and it doesnt save

**Comment by bradatkins** [comment]
Yes, I can enable it for the one...let me know if that's what you want

**Comment by bradatkins** [comment]
Try editing it again...

**Comment by chris206** [comment]
not able to save.
Take a look at my session

**Comment by chris206** [comment]
I just spent some time typing and submitting text or the Consulate email and it isnt there. I didnt save it of course, so that is frustrating.

Can you recover this info and fix this problem? The feature hasn't work since initiated and it is a really important idea
